### PR TITLE
1025 make sure translations used in the adopter fosterer scope

### PR DESCRIPTION
**Pull request by wuletawwonte**
# 🔗 Issue
#1025 
# ✍️ Description
- Plain strings were replaced in the adopter and fosterer scope with localized strings.
- Added new translation keys where necessary, ensuring consistency with the existing structure in the `en.yml` file.
# 📷 Screenshots/Demos

### CODE REVIEW

File: app/views/organizations/adopter_fosterer/adopted_pets/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <%= render DashboardPageComponent.new do |c| %>
-  <% c.with_header_title { "Adopted Pets" } %>
+  <% c.with_header_title { t(".header_title") } %>
```
Comment (kasugaijin) [inline app/views/organizations/adopter_fosterer/adopted_pets/index.html.erb:2]: Interesting…does Rails find the correctly scoped header title for this? I have not done it this way before.

File: app/views/organizations/adopter_fosterer/likes/index.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <%= render DashboardPageComponent.new do |c| %>
-  <% c.with_header_title { t('organizations.adopter_fosterer.likes.index.liked_pets') } %>
+  <% c.with_header_title { t('.header_title') } %>
```
Comment (kasugaijin) [inline app/views/organizations/adopter_fosterer/likes/index.html.erb:2]: Same question here

File: app/views/organizations/adopter_fosterer/adopted_pets/index.html.erb (hunk 1 above)
Comment (wuletawwonte) [reply app/views/organizations/adopter_fosterer/adopted_pets/index.html.erb:2]: Yes, Rails supports this, and it's called [Lazy Lookup](https://guides.rubyonrails.org/i18n.html#lazy-lookup). Rails automatically scopes the translation key based on the current controller and action, which is why I only need to use t(".header_title") here.

File: app/views/organizations/adopter_fosterer/likes/index.html.erb (hunk 2 above)
Comment (wuletawwonte) [reply app/views/organizations/adopter_fosterer/likes/index.html.erb:2]: https://github.com/rubyforgood/pet-rescue/pull/1037/#discussion_r1793249428

File: app/views/organizations/adopter_fosterer/adopted_pets/index.html.erb (hunk 1 above)
Comment (kasugaijin) [reply app/views/organizations/adopter_fosterer/adopted_pets/index.html.erb:2]: Very handy! Thanks for the explanation

**Review by kasugaijin — APPROVED** [review]
LGTM!

**Pull request merged**